### PR TITLE
refactor: #351 fix render images md, add styles to inputs, refactored…

### PR DESCRIPTION
**Pull request by Ekaterina1994**
… render ownWays, mentoringWays, favoritesWays

### CODE REVIEW

File: mw-webapp/src/utils/markdown/renderMarkdown.tsx [hunk 1]
```diff
@@ -26,18 +26,22 @@ const customComponents: Components = {
...
-          src={params.src}
-          alt={params.alt}
-          className={styles.markdownImg}
-        />
+        <div className={styles.imageContainer}>
+          <Image
+            src={params.src}
+            alt={params.alt}
+            className={styles.markdownImg}
+          />
```
Comment (tritonJS826) [inline mw-webapp/src/utils/markdown/renderMarkdown.tsx:43]: refactor!

Comment (Ekaterina1994) [reply mw-webapp/src/utils/markdown/renderMarkdown.tsx]: Done

File: mw-webapp/src/logic/userPage/UserPage.module.scss [hunk 2]
```diff
@@ -13,3 +13,20 @@
 .marginBottom {
   margin-bottom: $defaultMargin;
 }
+
+.inputH2 {
+  width: $minWidthBlock;
+  height: 27px;
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/userPage/UserPage.module.scss]: Created var

File: mw-webapp/src/component/title/Title.tsx [hunk 3]
```diff
@@ -33,6 +33,11 @@ interface TitleProps {
    */
   className?: string;
 
+  /**
+   * Additional style for input
+   */
+  styleInput?: string;
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/component/title/Title.tsx]: deleted

File: mw-webapp/src/component/editableTextarea/editableTextarea.tsx [hunk 4]
```diff
@@ -25,6 +25,11 @@ interface EditableTextareaProps {
    */
   className?: string;
 
+  /**
+   * Additional custom class name for the editable textarea
+   */
+  styleTextArea?: string;
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/component/editableTextarea/editableTextarea.tsx]: deleted

File: mw-webapp/src/component/editableTextarea/editableTextarea.tsx [hunk 5]
```diff
@@ -25,6 +25,11 @@ interface EditableTextareaProps {
    */
   className?: string;
 
+  /**
+   * Additional custom class name for the editable textarea
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/component/editableTextarea/editableTextarea.tsx]: Changed

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**